libsdhcdrivers/src/sdhc.c

```diff
@@ -205,7 +195,7 @@ UNUSED static void print_sdhc_regs(struct sdhc *host)
 {
     int i;
     for (i = DS_ADDR; i <= HOST_VERSION; i += 0x4) {
-        printf("%x: %X\n", i, readl(host->base + i));
+        printf("%x: %X", i, readl(host->base + i));
```

Comment:
can't we use ZF_LOGD() here also?

Reply:
I changed it to ZF_LOGD() with the latest commit. Since the function will probably only be used once someone starts debugging and wants to to dump the registers, it should be fine to replace the printf here.